beacon-chain/db/kv/blocks.go

```diff
@@ -354,6 +354,20 @@ func (k *Store) GenesisBlock(ctx context.Context) (*ethpb.SignedBeaconBlock, err
 	return block, err
 }
 
+// GenesisBlockRoot retrieves the genesis block root from db.
+func (k *Store) GenesisBlockRoot(ctx context.Context) ([32]byte, error) {
```

Comment:
we dont need this please revert this change.

Reply:
reverted. thanks